apps/emqx_redis/src/emqx_redis.app.src

```diff
@@ -0,0 +1,14 @@
+{application, emqx_redis, [
+    {description, "EMQX Enterprise Oracle Database Connector"},
```

Comment:
EMQX Enterprise Redis Database Connector"

Reply:
Thanks! Fixed in [5de6aed](https://github.com/emqx/emqx/pull/11139/commits/5de6aedc31956b49cc5af1dc5c6f273b8e55b94b)